src/scripts/keyboardNavigation.js

```diff
@@ -120,6 +121,15 @@ export function enable() {
                 inputManager.handleCommand('back');
                 break;
 
+            // HACK: Hisense TV uses Backspace for Back action
+            case 'Backspace':
+                if (browser.tv && browser.hisense) {
```

Comment:
Should this also check if there is no focus on an input like with the arrow keys?

Reply:
The user said that on-screen keyboard captures pressing `Back`, so we don't receive the click.

But now I am thinking about adding a check on the VIDAA OS, because Hisense could be Android, and it could have different behavior.

Also, I don't think it will work with a keyboard connected to the TV.